Dockerfile.dev

```diff
@@ -45,12 +46,23 @@ RUN apk update && apk upgrade && apk add tar \
     php7-mbstring \
     php7-memcached \
     php7-zlib \
+    php7-pear \
+    php7-dev \
     mysql-client \
     ssmtp \
     apache2 \
     ca-certificates \
     tzdata \
-    && rm -rf /var/cache/apk/*
+    php7-openssl
```

Comment:
Can all of the changes to the Dockerfiles be changed to simply adding `php7-mongodb` ?